ramls/record-bulk.raml

```diff
@@ -1,12 +1,12 @@
 #%RAML 1.0
-title: Inventory Storage Instance Bulk API
+title: Inventory Storage Record Bulk API
 version: v0.1
 protocols: [ HTTP, HTTPS ]
 baseUri: http://localhost
 
 documentation:
-  - title: Inventory Storage Instance Bulk Download API
-    content: <b>API for downloading a bulk set of Instance IDs</b>
+  - title: Inventory Storage records Bulk Download API
+    content: <b>API for downloading a bulk set of records IDs</b>
```

Comment:
just for readability sake, please add what records can be downloaded and that ID is infact the UUID of the records